pkg/operator/status_test.go

```diff
@@ -81,27 +92,44 @@ func TestOperatorStatusProgressing(t *testing.T) {
 		optr.osClient = fakeconfigclientset.NewSimpleClientset(co)
 		startTime := metav1.Now()
 		optr.statusProgressing()
-		o, _ := optr.osClient.ConfigV1().ClusterOperators().Get(clusterOperatorName, metav1.GetOptions{})
+		got, _ := optr.osClient.ConfigV1().ClusterOperators().Get(clusterOperatorName, metav1.GetOptions{})
```

Comment:
Why is it `got` now?

Reply:
`got` does not seem very intuitive. may be `co`?

Reply:
I usually use `got` vs `expected`, I don't mind, if you find this a blocker for the PR please Iet me know so I'll put it back or whatever the majority prefer

Reply:
in that case may be `gotCo`, just like "Condition" in `expectedCondition`.
not a blocker though.